app/src/test/java/ryey/easer/skills/usource/location/LocationUtilsTest.java

```diff
@@ -0,0 +1,13 @@
+package ryey.easer.skills.usource.location;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class LocationUtilsTest {
+    @Test
+    public void When_locationIsNull_IsAcceptable_Should_ReturnFalse(){
```

Comment:
Maybe change the method name to follow the method name convention of *camelCase*?

Reply:
Done. Thanks for the review